ntc_templates/templates/cisco_xr_show_bgp_vrf_all_neighbors_advertised-routes.textfsm

```diff
@@ -0,0 +1,20 @@
+Value NETWORK (\d+\.\d+\.\d+\.\d+)
+Value PREFIX_LENGTH (\d+)
+Value NEXT_HOP (\d+\.\d+\.\d+\.\d+)
+Value FROM ((\d+\.\d+\.\d+\.\d+)|\S+)
+Value AS_PATH (.*)
```

Comment:
```suggestion
Value AS_PATH (.+)
```

AS Path should contain something so switching this from zero or more to one or more characters.

If you approve of this edit, please use the button in the GitHub web interface to accept my suggestion as this gives credit and blame if there were a problem. :grin: